### PR TITLE
[DOC] use new surface API in plot_surf_atlas

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes none but relates to #4568

Changes proposed in this pull request:

- part of moving surface API to stable codebase
- replace plot_surf_atlas with example taken from experimental

### CODE REVIEW

File: nilearn/experimental/surface/_datasets.py [hunk 1]
```diff
@@ -116,29 +116,3 @@ def fetch_nki(mesh_type: str = "pial", **kwargs) -> Sequence[SurfaceImage]:
         images.append(img)
 
     return images
-
-
-def fetch_destrieux(
```
Comment (Remi-Gau) [inline nilearn/experimental/surface/_datasets.py:121]: remove this "shortcut" to make it explicit in examples how we construct the SurfaceImage: [#4642](https://github.com/nilearn/nilearn/issues/4642)

**Review by bthirion — APPROVED** [review]
LGTM from looking at the changes, but the doc has not built yet ... to be checked...

**Comment by Remi-Gau** [comment]
blocked by #4674 
rebased after it is merged

File: doc/Makefile [hunk 2]
```diff
@@ -108,6 +108,12 @@ html-noplot: sym_links_datasets glm_reports
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
+html-noplot-noreport: sym_links_datasets
```
Comment (bthirion) [inline doc/Makefile]: Is this related to this PR ?

**Review by bthirion — COMMENTED** [review]
LGTM overall !

File: doc/Makefile (hunk 2 above)
Comment (Remi-Gau) [reply doc/Makefile]: nope: needed a faster way to build the doc locally (this does not generate the html report for maskers and glm , that usually takes some extra minutes)

**Pull request merged**